api/types/events/api.go

```diff
@@ -71,10 +71,24 @@ type AuditEvent interface {
 	SetClusterName(string)
 }
 
+type AuditContext struct {
+	context.Context
+}
+
+func BackgroundAuditCtx() AuditContext {
```

Comment:
nit: Spell out Context?

```suggestion
func BackgroundAuditContext() AuditContext {
```